velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -169,7 +169,7 @@ std::shared_ptr<const ParquetTypeWithId> ReaderBase::getParquetColumnInfo(
               maxDefine);
         }
         case thrift::ConvertedType::MAP_KEY_VALUE: {
-          // child of MAP
+          // children of MAP
```

Comment:
This is actually "child" because MAP_KEY_VALUE node is the single child of a MAP node. Note that the Parquet schema has extra levels under MAP or LIST. The child of MAP type node is a MAP_KEY_VALUE type node, and it has 2 children. It's not like the 2 children are directly from a MAP node.